learning-outcomes.md

```diff
@@ -0,0 +1,58 @@
+# Week 1 Learning Outcomes
+
+## Teamwork & Toolkit
+
+### Pair programming
+
+- [ ] **We can use pair programming techniques to work together**
+- [ ] We can work on one computer as a pair
+- [ ] We can use four different paired programming techniques
+- [ ] We can distribute commits evenly among team members
+
+---
+
+### Version control
+
+- [ ] **We can use Git and GitHub as version control for our code**
+- [ ] We can initialise a Git repository so that we can track changes to our code over time
+- [ ] We can add / stage files so that we can include these updates in our next commit
+- [ ] We can write descriptive commit messages so that our commit history tells a story
+- [ ] We can amend commit messages so that we can correct typos/mistakes before we push to GitHub
+- [ ] We can co-author commit messages so that our commit history reflects everyone’s contributions
+- [ ] We can merge master to our feature branch so that we can resolve any merge conflicts locally
+- [ ] We can resolve merge conflicts to maintain a working codebase
+- [ ] We can push and pull branches from a remote to work on different tasks
+- [ ] We can create a pull request on GitHub to request feedback from our teammates
+- [ ] We can review other people’s pull requests to provide feedback on their code
+- [ ] We can delete branches that have been merged to keep our repo tidy
+
+---
+
+### Accessibility
+
+- [ ] **We can make a website that is accessible by people other than ourselves**
+- [ ] We can structure a site using semantic HTML to aid accessibility
+- [ ] We can add appropriate attributes to HTML elements to make them screen reader friendly
+- [ ] We can add ARIA labels (where necessary) to make our semantic HTML more accessible
+- [ ] We can run a Lighthouse accessibility audit to check our website’s accessibility
+- [ ] We can use a screen reader to understand how people with visual impairments maye experience our website
+- [ ] We can implement a suitable contrast ratio on our website so that it meets the needs of people who are colour blind
```

Comment:
I think these are two related but separate concepts. Higher contrast benefits everyone, especially people with visual impairments. Colour-blindness is more affected by the hues chosen rather than contrast levels.

```suggestion
- [ ] We can design a UI without relying solely on colour, so that we don't exclude colour-blind users
- [ ] We can ensure our UI has sufficient colour contrast so that everyone can perceive it comfortably
```

Reply:
Have accepted your suggested change.